threenums.c

```diff
@@ -0,0 +1,23 @@
+/*Write a program threenums.c that takes in 3 user-input numbers and prints “numbers are equal” if all 3 numbers are equal, and “not equal”, followed by the largest number, if the numbers are not equal. Then print out the sum of the 3 numbers.*/
+#include <stdio.h>
+int main(){
+	int x,y,z;
+	printf("Please give me three numbers: \n");
+	scanf("%d %d %d", &x, &y, &z);
+/*	printf("your numbers are: %d %d %d", x,y,z);*/
+
+	if (x==y && y==z)	
+		printf("numbers are equal\n");
+	else 
+		printf("The numbers are not equal, the largest number is");
+if (!(x==y && y==z));{
```

Comment:
your code printed out the largest number regardless of whether they were equal or not and the reason for this is because you have this semicolon before the {. be careful - this returned a warning.